recipe/meta.yaml

```diff
@@ -30,11 +30,14 @@ requirements:
     - setuptools >=51.0.0
     - cython >=0.29.24,<3
     - numpy
+    - versioneer
+    - tomli  # [py<311]
   run:
     - python
     - {{ pin_compatible('numpy') }}
     - python-dateutil >=2.8.1
     - pytz >=2020.1
+    - python-tzdata >=2022a
```

Comment:
BTW, tzdata is included in modern Python but pip is designed to find it only if installed by the python alternative like this. We could probably drop the `pip check` test too. But "overi-nstalling" tzdata is not a big deal and probably better than dropping the pip-check.